github_pr.py

```diff
@@ -42,6 +43,8 @@ def _print_pr(pr, **args):
     elif 'comments' in args and args['comments']:
         for comment in pr.get_comments():
             print "#%d : Comment - %s" % (pr.number, comment.body)
+    elif 'filters' in args and args['filters']:
+        print "%d" % (pr['pr'].number)
```

Comment:
I don't like this, why would we just print the number? The other options in here are around limiting what the output looks like. Just because we have filters doesn't mean we should show only some things. I'd still like to be able to have a tabular output when using filters.